Make error Fedora 24

when I'm typing make I've got this error :
```
cc -std=c99 -Iinclude -pedantic -Wall -Os `pkg-config --cflags pango pangocairo libconfig
gdk-pixbuf-2.0` -DVERSION=\"0.4.0-42-gb583b34\" -D_POSIX_C_SOURCE=199309L -DYA_INTERNAL -DYA_DYN_COL -DYA_ENV_VARS -DYA_INTERNAL_EWMH -DYA_ICON -DYA_NOWIN_COL -DYA_MUTEX -DYA_VAR_WIDTH  -c -o src/ya_parse.o src/ya_parse.c
Package gdk-pixbuf-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `gdk-pixbuf-2.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'gdk-pixbuf-2.0' found
In file included from src/ya_parse.c:9:0:
include/yabar.h:24:25: fatal error : pango/pango.h : No such file or directory
 #include <pango/pango.h>
                         ^
compilation terminée.
<commande interne> : la recette pour la cible « src/ya_parse.o » failed
make: *** [src/ya_parse.o] Erreur 1
```

I have verified that pongo is installed and it's version by running pkg-config :
```
$ pkg-config --modversion pango
1.40.1
```
any idea to solve this issue ?

--- a/README.md
+++ b/README.md
@@ -30,7 +30,10 @@ Yabar is a modern and lightweight status bar that is intended to be used along w
 A Yabar session should contain one or more *bars* within the same session. Each bar should contain one or more *blocks*. Each block should display some useful info to the user (free memory, CPU temperature, etc...).
 
 ## Installation
-Yabar initially requires libconfig, cairo and pango. The feature `DYA_INTERNAL_EWMH` in `Makefile` additionaly xcb-ewmh (or xcb-util-wm in some distros) and the feature `-DYA_ICON` requires gdk-pixbuf. These dependencies can be installed through your distribution's package manager, such as `dnf install libconfig-devel cairo-devel pango-devel` on Fedora or `sudo apt-get install libcairo2-dev libpango1.0-dev libconfig-dev libxcb-randr0-dev libxcb-ewmh-dev` on Ubuntu.
+Yabar initially requires libconfig, cairo and pango. The feature `DYA_INTERNAL_EWMH` in `Makefile` additionaly xcb-ewmh (or xcb-util-wm in some distros) and the feature `-DYA_ICON` requires gdk-pixbuf2. These dependencies can be installed through your distribution's package manager:
+
+* Fedora: `dnf install libconfig-devel cairo-devel pango-devel gdk-pixbuf2-devel`
+* Debian / Ubuntu: `apt-get install libcairo2-dev libpango1.0-dev libconfig-dev libxcb-randr0-dev libxcb-ewmh-dev libgdk-pixbuf2.0-dev`
 
 You can install yabar as follows:
 
@@ -39,7 +42,9 @@ You can install yabar as follows:
 		$ make
 		$ sudo make install
 
-If you use libconfig 1.4.x (still used in Ubuntu 14.04 and Debian), please type `export CPPFLAGS=-DOLD_LIBCONFIG` then build using `make` as usual.
+If you use libconfig 1.4.x (still used in Ubuntu 14.04 and Debian Jessie), please type `export CPPFLAGS=-DOLD_LIBCONFIG` then build using `make` as usual.
+
+For Arch Linux users there are AUR packages available: [yabar](https://aur.archlinux.org/packages/yabar/) and [yabar-git](https://aur.archlinux.org/packages/yabar-git/)
 
 ## Configuration
 

